lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+module Players
+  class Human < Player
+
+    def move(board)
+      puts "Please enter 1-9:"
+      return gets.strip
+    end
```

Comment:
no need to use the `return` keyword, Ruby convention is to avoid it unless it's explicitly needed